tdrs-frontend/src/components/Footer/Footer.jsx

```diff
@@ -20,6 +22,18 @@ function Footer() {
                     Privacy policy
                   </a>
                 </li>
+                {authenticated ? (
+                  <li className="mobile-lg:grid-col-6 desktop:grid-col-auto usa-footer__primary-content">
```

Comment:
I never used mobile-lg. We should probably look through the frontend together at some point to get our styling as consistent as possible. Maybe we can save something like that for when we don't have such a big deadline.

Reply:
I changed the footer in my pr too 
https://github.com/raft-tech/TANF-app/pull/1924/files#diff-9fabec8f76eaeefca82056f5b7fb1288c54eaaa656dcfcb54473eb4413102796